### PR TITLE
ncap project

### PR DESCRIPTION
**Pull request by glibesyck**

### CODE REVIEW

File: projects/project-notebooks/NeuralCircuitArchitecturalPriors.ipynb [hunk 1]
```diff
@@ -0,0 +1,2365 @@
...
...c1LDUs14MD1oiVtIZfbLA8JUY2JsFyrAhYHRObcxQS19a0zTimx51lWVZy2Oq+32M5jcm48j68xzqZv4mFld08haJp92shcXtusn5joZEtuyJgtYWmbgBiWZZlJY5Z3Zt9lqvEablWeCzPcqflWNZg9SVmGLxPRqwjzzdZ/js9lud2n9Xw1KB17BmfZUxsSHPRezLr5K7cPWX4rO4zubn89R6rsXIyxrC7LP/2QetU3iYfJ/c15eIXCwzLeWeT1RdLFzg3078PCmxzb6PlKZ78jvLtGLT2POi0jNWTMXRqb8Pkd8RCt9Wwt89qcc++GccCy7KsgpKTIiIiIiIi8ifOJHU6wf/f3v2DRhFEcRz/FSm2SLEHKbJygeyhRdJ5YJGzDAgWFmqlgkUuNgqCJNgYLNTKBAVJLCTGRlNqIaZTwcJYJYWYKzfgwlgIN0XAV1xhk5iYuJdL7jxh/X7a/TMz7JspHvNm60GkuL8zxaart4ZVWa5q7d3E76Vo5pWsOymM22trwylJTcFgrOgwr/GJal9N4cCQot27rswp8YWm/bM0UbIhRUfj7d1Le27yStK6gr54bxttM/nUyVmgqBgpzOqqmRRsX7TX4ypddLqbvFF166y7dr6Jd6p9M4X92WP0aaJ6o6BoMDxgKbPJpaaouOvF+7bZQjxvjbkvVtzXqQLrzXZ7Cop397mTzCtJf6gwGGXHXtajrcTthlOS+tbioeHl1p2sJ2r6fc1MwY44/HhzWKOfqlr7sGN9aDYnWxmXSVEx/vNcaJhc6mSHWeMy1oN922xl/m/FcjFW1JtxzyYSfQAAAACArlt9Nqmn71e19KqmypM1Pb/wFxMeaG55SuVTi4pvz2rq7HGF9WXNTYxroXdaX5aqrfxfGGhb7eGoTt6Xrj6eUfVEKPu8qMmxe3LXV7Ty62xI7Icz+gAAAAAA3dcwKRzRtYW3mifJ92+NTGr2TkXJg/MqHzuiUuWSXvaMaf4RST50z9CVGU2fMS1eLqs0UNLwuTn507N6cYMk30Gwow8AAAAAAEiSzHtZEGaX+AJd4L97BWGooAN/wP3fkOgDAAAAAAAAcoDSXQAAAAAAACAHSPQBAAAAAAAAOUCiDwAAAAAAAMgBEn0AAAAAAABADpDoAwAAAAAAAHKARB8AAAAAAACQAz8BBTBCUps55uYAAAAASUVORK5CYII=)"
```
Comment (iamzoltan) [inline projects/project-notebooks/NeuralCircuitArchitecturalPriors.ipynb:119]: why is this image like this?

File: projects/project-notebooks/NeuralCircuitArchitecturalPriors.ipynb [hunk 2]
```diff
@@ -0,0 +1,2365 @@
...
+    "env.action_spec()"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "id": "jIWY7WBpuM_8"
+   },
+   "source": [
+    "## 1.4 Example of simple modific
```
Comment (iamzoltan) [inline projects/project-notebooks/NeuralCircuitArchitecturalPriors.ipynb:1222]: same here

Comment (iamzoltan) [inline projects/project-notebooks/NeuralCircuitArchitecturalPriors.ipynb:2209]: why is there widget state?

File: projects/project_materials.yml [hunk 3]
```diff
@@ -7,19 +7,15 @@
...
-    title: Example Past Project
-  # You should add your past pojects here using the following:
-  # - file: point to the file
-  #   title: name the project
\ No newline at end of file
+# - file: projects/docs/past_projects_overview.md
+#   title: Past projects
+#   sections:
+#   - file: projects/past-projects/example_past_project.ipynb
+#     title: Example Past Project
```
Comment (iamzoltan) [inline projects/project_materials.yml:21]: looks good

**Review by iamzoltan — CHANGES_REQUESTED** [review]
Looks like there are artifacts from colab. Can you try to clean them up?

There also seems to be some issues with images. How are you embedding them?

**Comment by glibesyck** [comment]
added & updated in #148.

**Pull request closed without merging**